specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/webapplicationfirewall.json

```diff
@@ -329,6 +329,10 @@
             "name": "WebApplicationFirewallPolicyResourceState",
             "modelAsString": true
           }
+        },
+        "botDetection": {
+          "description": "Describes bot detection for policy.",
```

Comment:
sounds a little odd. Do you mean something more like describes bot detection settings for the policy?